tests/src/JIT/SIMD/Vector3Interop.cs

```diff
@@ -37,6 +52,20 @@ class PInvokeTest
         Vector3 v2);
 
     [DllImport(@"Vector3TestNative", CallingConvention = CallingConvention.StdCall)]
+    public static extern float nativeCall_PInvoke_Vector3Arg_Unix(
+        Vector3 v3f32_xmm0, 
+        float f32_xmm2, 
+        float f32_xmm3,
+        float f32_xmm4, 
+        float f32_xmm5, 
+        float f32_xmm6, 
+        float f32_xmm7,
+        float f32_mem0, 
+        Vector3 v3f32_mem1, 
+        float f32_mem2, 
+        float f32_mem3);        
```

Comment:
Since a bug was revealed in passing Vector3 stack arg in RPInvoke scenario, please consider adding a separate test case (both PInvoke and RPInvoke) with another Vector3 stack arg after f32_mem.